xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamPVRManager.h

```diff
@@ -120,7 +120,7 @@ class CDVDInputStreamPVRManager
   bool m_isOtherStreamHack;
   PVR_STREAM_PROPERTIES *m_StreamProps;
   std::map<int, std::shared_ptr<CDemuxStream>> m_streamMap;
-  bool m_isRecording;
+  bool m_isRecording = false;
```

Comment:
true wouln't make any differernce here. this is inititialization for the sake of initialization. I don't get the benefit of this. a ready may think that this has influence on behavious but it has not.